collect_scripts.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+# Copyright 2021 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import shutil
+import sys
+from pathlib import Path
+from stat import S_IREAD, S_IWUSR, S_IWGRP, S_IRGRP, S_IEXEC
+
+FILE_EXTENSIONS = [".py", ".sh", ".json", ".pl", ".obo"]
+
+IGNORE_EXTENSIONS = [
+    ".cwl",
+]
+
+IGNORE_LIST = [
+    "__init__.py",
+    "__init__",
+    "copy_utils.py",
+]
+
+# binaries or scripts with no extension
+SPECIFIC_FILES = [
+    "add_header",
+    "awk_tool",
+    "owltools",
+    "faselector",
+    "format_bedfile",
+    "trimmomatic",
```

Comment:
I think we should add '.sh' extensions to add_header, awk_tool, format_bedfile and trimmomatic

Reply:
Yeah, I'll do that :).